R/check_package.R

```diff
@@ -2,7 +2,7 @@
 #'
 #' This function perform checks on a package before publishing
```

Comment:
Add a bit more info about specific checks here (pids, etc.) but otherwise the function looks excellent (and readable!)